<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ROIConfigurationWidget</class>
 <widget class="QDialog" name="ROIConfigurationWidget">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ROI Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lbWebcam">
         <property name="text">
          <string>Webcam</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cbVideoDevice">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBoxDetectionType">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Detection type</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="topMargin">
           <number>6</number>
          </property>
          <item row="1" column="1">
           <widget class="QComboBox" name="cbColorIs">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>is</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>is not</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cbOperator">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>&gt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>&lt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>=</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>â‰ </string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="cbColorDT">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="dspNumberDT">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <double>-10000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999.000000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="cbSymbolIs">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>is</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>is not</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QComboBox" name="cbTextFont">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Auto</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PR4G</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rbNumberDT">
            <property name="text">
             <string>Number</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QRadioButton" name="rbUndefinedDT">
            <property name="text">
             <string>Undefined</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rbColorDT">
            <property name="text">
             <string>Color</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="lbFiltration">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Filtering</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="sbFiltration">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="lbNumberFont">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Font :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QComboBox" name="cbNumberFont">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
            <item>
             <property name="text">
              <string>Auto</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Digital 7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PR4G</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rbTextDT">
            <property name="text">
             <string>Text</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="cbTextIs">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>is</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>is not</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="lbTextFont">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Font :</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="leTextToDetect">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="4">
           <widget class="QLabel" name="lbTolerancePercentage">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>100%</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="3">
           <widget class="QSlider" name="sTolerance">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="lbTolerance">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Tolerance</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QRadioButton" name="rbSymbolDT">
            <property name="text">
             <string>Symbol</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="pbSetSymbol">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Set symbol</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2" rowspan="2" colspan="3">
           <widget class="QLabel" name="lbSymbol">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <layout class="QGridLayout" name="gridLayout_13"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Trigger alarm when...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>200</width>
         <height>55</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Edit ROI</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item alignment="Qt::AlignLeft">
             <widget class="QLabel" name="lbROIX">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>X :</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QSpinBox" name="sbROIX">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="maximum">
               <number>1920</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>60</number>
            </property>
            <item alignment="Qt::AlignLeft">
             <widget class="QLabel" name="lbROIWidth">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Width :</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QSpinBox" name="sbROIWidth">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="maximum">
               <number>1920</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item alignment="Qt::AlignLeft">
             <widget class="QLabel" name="lbROIY">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Y :</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QSpinBox" name="sbROIY">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="maximum">
               <number>1080</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="leftMargin">
             <number>60</number>
            </property>
            <item alignment="Qt::AlignLeft">
             <widget class="QLabel" name="lbROIHeight">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Height :</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QSpinBox" name="sbROIHeight">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="maximum">
               <number>1080</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbROI">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>268</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOK">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="cwVideoPlayer" name="videoPlayer" native="true"/>
   </item>
  </layout>
  <zorder>videoPlayer</zorder>
  <zorder></zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cwVideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">cwvideoplayer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbOK</sender>
   <signal>clicked()</signal>
   <receiver>ROIConfigurationWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>ROIConfigurationWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>850</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
